src/k8s/cmd/k8s/k8s.go

```diff
@@ -11,9 +12,13 @@ var (
 	rootCmdOpts struct {
 		logDebug   bool
 		logVerbose bool
+		stateDir   string
 	}
+	k8sdClient client.Client
+)
 
-	rootCmd = &cobra.Command{
+func NewRootCmd() *cobra.Command {
+	rootCmd := &cobra.Command{
```

Comment:
do we still need the `New$command()` and `new$COmmand()`? I'm okay with it, or let's remove if you don't think it's useful now.

Reply:
Let's keep it - we can use it to have isolated unit-tests later.